classes/PrestaShopBackup.php

```diff
@@ -42,7 +42,7 @@ class PrestaShopBackupCore
     public $customBackupDir = null;
 
     /** @var bool|string */
-    public $psBackupAll = true;
+    public $psIgnoreStats = false;
```

Comment:
You're introducing a Backward Incompatible Change by renaming a variable. Would it be possible to avoid that?

Solution 1: don't rename the variable
Solution 2: keep both old and new variable

Reply:
Do you refer to the name of the variable stored in **ps_configuration** table ?
It's fine I can retain the old name, but then it would have the wrong meaning - I don't like it 🤔

I have already verified that PS_BACKUP_ALL is not called anywhere else in the codebase, the right thing to do would be to copy the old => new value at PS version upgrade, but I wouldn't know how to do that.

Reply:
So let's do it right by keeping both variables , migrating the old one to the new one during PS upgrade. 
IIs there any KB / doc that can show me the way ? I have never explored the installer so far.

Reply:
Hi @gennaris 

Do you think you can ensure this PR doesn't have any BC Breaks?

Reply:
@kpodemski Worst case scenario is that during a PS upgrade the merchant would have
the "Ignore Statistics" toggle reseted unless the old PS_BACKUP_ALL value
is getting migrated in the new PS_IGNORE_STATS value during the upgrade phase,
since here I am renaming a configuration variable to give it the correct wording/meaning,
(I think it's the right thing to do)
So, as per my previous message : 
Is there any KB / doc that can show me the way ? I have never explored the installer so far.
This way I can complete the PR.